Samples/V1.0Samples/RemoteMediaSamples/IncidentBot/README.md

```diff
@@ -58,6 +58,118 @@ The sample demostrate an incident process workflow. When a incident raised (thro
 
 5.	API references
 
```

Comment:
We might need some description of ADF directory in this readme doc

Reply:
Added under Script internal for both this PR and AVBot PR